packages/liferay-npm-scripts/src/utils/filterChangedFiles.js

```diff
@@ -27,6 +33,23 @@ function filterChangedFiles(files) {
 
 	const mergeBase = git('merge-base', 'HEAD', upstream);
 
+	// Check for changes in liferay-npm-scripts version.
+	try {
+		git(
+			'diff',
+			mergeBase,
+			'-Gliferay-npm-scripts',
+			'--quiet',
+			'--',
+			'modules/package.json',
+			'modules/private/package.json'
+		);
+	} catch (error) {
+		if (error.toString().includes('exited with status 1.')) {
```

Comment:
Note that we are checking for "status 1." (see the full-stop here). This is to avoid fails positives, like if it exits with a status code of 129 (which can happen if `git-diff` is invoked improperly).

Reply:
Further reading:

- https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---quiet
- https://git-scm.com/docs/git-diff#Documentation/git-diff.txt---exit-code

I might add this as a code comment, as GitHub comment is probably too ephemeral.